tabrepo/benchmark/task/openml/task_wrapper.py

```diff
@@ -155,3 +161,18 @@ def subsample_combined(
     ) -> pd.DataFrame:
         data, _ = self.subsample(X=data, y=data[self.label], size=size, random_state=random_state)
         return data
+
+
+class OpenMLS3TaskWrapper(OpenMLTaskWrapper):
+    """
+    Class which uses S3 cache to download task splits.
+    """
+    def __init__(self, task, s3_dataset_cache: str = None):
+        super().__init__(task)
+        self.s3_dataset_cache = s3_dataset_cache
```

Comment:
No need to have `__init__`? What is the point of setting `self.s3_dataset_cache`?